### ISSUE TITLE
Migrate eVSS calls to rated disabilities functionality (vets-api /rated_disabilities) - test integration on DEV, SANDBOX, STAGING

### ISSUE THREAD

**Issue by ahay-agile6** [department-of-veterans-affairs/va.gov-team#56766]
This is a follow up to #54161.

There was something wrong with Lighthouse's authentication in DEV, SANDBOX, and STAGING for the Veteran Verification APIs where they need to do a code change to fix it.

They need to do a code change and then let us know.

Once this happens, we can test the Rate Disabilities endpoint in staging and production.

**Update** from LH PM Matthew Burnett - LH API Vet Verif, V2 does include a rated disability endpoint. (new API name: Veteran Service History & Eligibility, V2). Has been blocked by VA Profile but the removal of that dependency is in progress, available in Sandbox by 5/18.

### Acceptance Criteria:
- Identify if there are any authentication errors when connecting to LH for each environment (Dev, sandbox and staging)
- Resolve errors in the sprint, if possible 
- Create another ticket to resolve authentication errors, if needed

**Comment by RakshindaAslam** [comment]
cc @RakshindaAslam

**Comment by RakshindaAslam** [comment]
@ahay-agile6 - is there a slack thread on this?

**Comment by sethdarragile6** [comment]
Verified that the relevant env vars were populated in dev, sandbox and staging. Sandbox example:

> LIGHTHOUSE_FORM526_VETERAN_VERIFICATION_HOST=https://sandbox-api.va.gov
LIGHTHOUSE_FORM526_VETERAN_VERIFICATION_AUD_CLAIM_URL=https://deptva-eval.okta.com/oauth2/ausi3u00gw66b9Ojk2p7/v1/token
LIGHTHOUSE_FORM526_VETERAN_VERIFICATION_ACCESS_TOKEN_CLIENT_ID=0oalg6awo4RAPumSp2p7
LIGHTHOUSE_VETERAN_VERIFICATION_HOST=https://sandbox-api.va.gov
LIGHTHOUSE_VETERAN_VERIFICATION_AUD_CLAIM_URL=https://deptva-eval.okta.com/oauth2/ausi3u00gw66b9Ojk2p7/v1/token

Also validated that calls are going out to LH, though we do have a standing issue at the moment where responses are 404.
